### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cmvinod**
Modified original VGA simulator link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Simulate display output of a VGA core by running a testbench.
 
-Original VGA Simulator: https://www.ericeastwood.com/lab/vga-simulator/
+Original VGA Simulator: [https://www.ericeastwood.com/blog/vga-simulator-getting-started/](https://ericeastwood.com/blog/vga-simulator-getting-started/)
```
Comment (benjamin051000) [inline README.md:5]: You don't need to use the \[\]\(\) syntax here. Just replace it with the new link on its own: "https://ericeastwood.com..." without the brackets/parens.

**Review by benjamin051000 — CHANGES_REQUESTED** [review]
Thanks for the update. Please see the comment I made in the review, just a minor tweak. Once that is fixed, looks good to merge.

File: README.md (hunk 1 above)
Comment (cmvinod) [reply README.md]: Thank you for the suggestion. I made appropriate changes. Kindly review them. This would be my first commit. Thank you for the support.

**Review by cmvinod — COMMENTED** [review]
Thank you for the suggestion. I made appropriate changes. Kindly review them. This would be my first commit. Thank you for the support.

**Review by benjamin051000 — APPROVED** [review]
looks good

**Pull request merged**